Mesh_3/examples/Mesh_3/CMakeLists.txt

```diff
@@ -149,6 +149,11 @@ if(TARGET CGAL::CGAL_ImageIO)
     target_link_libraries(mesh_3D_image_with_custom_initialization
                           PUBLIC CGAL::Eigen3_support)
 
+    create_single_source_cgal_program(
+      "mesh_3D_gray_image_with_custom_initialization.cpp")
+    target_link_libraries(mesh_3D_image_with_custom_initialization
+                          PUBLIC CGAL::Eigen3_support)
```

Comment:
@lrineau isn't that linking instruction correct?

Reply:
It is correct for `mesh_3D_image_with_custom_initialization`. :smile: